pkg/kn/commands/source/apiserver/describe.go

```diff
@@ -110,10 +109,10 @@ func writeResources(dw printers.PrefixWriter, apiVersionKindSelectors []v1alpha2
 
 func writeSink(dw printers.PrefixWriter, sink duckv1.Destination) {
 	subWriter := dw.WriteAttribute("Sink", "")
-	subWriter.WriteAttribute("Name", sink.Ref.Name)
-	subWriter.WriteAttribute("Namespace", sink.Ref.Namespace)
 	ref := sink.Ref
 	if ref != nil {
+		subWriter.WriteAttribute("Name", sink.Ref.Name)
+		subWriter.WriteAttribute("Namespace", sink.Ref.Namespace)
 		subWriter.WriteAttribute("Kind", fmt.Sprintf("%s (%s)", sink.Ref.Kind, sink.Ref.APIVersion))
```

Comment:
Should this be `"Resource"` instead? Every other source describe command uses `"Resource"` except apiserver.

Reply:
I can comment out lines 120 and 122 and still pass tests...

```go
func writeSink(dw printers.PrefixWriter, sink duckv1.Destination) {
	subWriter := dw.WriteAttribute("Sink", "")
	ref := sink.Ref
	if ref != nil {
		subWriter.WriteAttribute("Name", sink.Ref.Name)
		subWriter.WriteAttribute("Namespace", sink.Ref.Namespace)
		subWriter.WriteAttribute("Kind", fmt.Sprintf("%s (%s)", sink.Ref.Kind, sink.Ref.APIVersion))
	}
	uri := sink.URI
	// if uri != nil {
		subWriter.WriteAttribute("URI", uri.String())
	// }
}
```

Reply:
That would make sense wouldn't it? 

This would be the output of `TestSimpleDescribe`:

```
Actual output: Name:                testsource
        Namespace:           
        Age:                 
        ServiceAccountName:  testsa
        EventMode:           Reference
        
        Sink:         
          Name:       testsvc
          Namespace:  default
          Kind:       Service (serving.knative.dev/v1)
          URI:        
        
        CloudEvent Overrides:  
          foo:                 bar
        Resources:             
          Kind:                Event (v1)
        
        Conditions:  
          OK TYPE    AGE REASON
```

Basically, `uri.String()` is nil protected. The point of the `if` is to not display `URI`, which it will not if the `if uri != nil` is left in place.

Reply:
Added a ContainsNone assertion to make sure `URI` is not present when using Sink Ref in test cases.